mtenv/wrappers/multitask.py

```diff
@@ -51,8 +48,20 @@ def assert_task_seed_is_set(self) -> None:
         assert self.np_random_task is not None, "please call `seed_task()` first"
         self.env.assert_task_seed_is_set()
 
-    def reset(self) -> ObsType:
-        return self.env.reset()
+    def reset(  # type: ignore[override]
+        self,
+        *,
+        env_seed: Optional[int] = None,
+        task_seed: Optional[int] = None,
+        return_info: bool = False,
+        options: Optional[Dict[str, Any]] = None,
+    ) -> Union[ObsType, Tuple[ObsType, InfoType]]:
+        return self.env.reset(
+            env_seed=env_seed,
+            task_seed=task_seed,
+            return_info=return_info,
+            options=options,
+        )
```

Comment:
Why bother to repeat all arguments instead of using something like `reset(self, **kwargs)` and forwarding them to the sub-env? (an approach you used in other places)